tests/unit/test_yamlparser.py

```diff
@@ -120,6 +120,7 @@ def test_encoder_name_env_replace():
 
 
 def test_encoder_name_dict_replace():
+    os.environ['BE_TEST_NAME'] = 'hello123'
```

Comment:
can u please use a fixture to set the env variable and reset afterwards? this is done in many other places